changelogs/changelog.yaml

```diff
@@ -3,70 +3,70 @@ releases:
   1.1.0:
     changes:
       bugfixes:
```

Comment:
I am afraid we need to update this one after every release as this is a generated file.
Could you please add it as excluded in the `ansible-lint.yml` configuration file ?

Reply:
It looks like we shouldn't need to exclude it with the production profile, but for some reason the action [doesn't seem to be using that profile from the config](https://github.com/ansible-collections/cloud.terraform/actions/runs/6826868113/job/18567718986?pr=102#step:3:254)...any idea why?

Reply:
Maybe I'm missing something, but I don't see `changelogs/changelog.yaml` in `.ansible-lint`. It passes locally when I add it.

Reply:
I haven't added it yet. I will, but even without it in .ansible-lint it passes if the production profile is used. My bigger concern is that it doesn't seem like the GHA workflow is using that profile?